ocs_ci/utility/rhv.py

```diff
@@ -1,18 +1,81 @@
 import logging
+import os
+import time
+import yaml
+
+import ovirtsdk4 as sdk4
 
 logger = logging.getLogger(__name__)
 
+# default location of files with necessary ovirt cluster details
+OVIRT_CONFIG_FILEPATH = os.path.expanduser("~/.ovirt/ovirt-config.yaml")
+
+
+class RHVTimeoutException(Exception):
+    pass
+
+
+def load_ovirt_config(conf_path=None):
```

Comment:
suggest to follow the convention of other platforms, in which we have class with the platform's API related methods

Reply:
removing this function and handling it via data config